toolchain/sem_ir/formatter.cpp

```diff
@@ -125,23 +231,27 @@ class FormatterImpl {
   }
 
   // Formats a top-level scope, particularly Constants and ImportRefs.
-  auto FormatScope(InstNamer::ScopeId scope_id, llvm::ArrayRef<InstId> block)
-      -> void {
+  auto FormatScope(InstNamer::ScopeId scope_id, llvm::ArrayRef<InstId> block,
+                   bool tentative = false) -> void {
```

Comment:
I think there are just two calls to this, and both pass `tentative=true`. Had you considered just making it the standard behavior?

```suggestion
  auto FormatScope(InstNamer::ScopeId scope_id, llvm::ArrayRef<InstId> block)
      -> void {
```

Reply:
I think this is a somewhat surprising default, but maybe with a rename of the function that can be addressed. I'll take a look.

Reply:
Done